src/dg/dg.cpp

```diff
@@ -1257,6 +1407,14 @@ void DGBase<dim,real,MeshType>::assemble_residual (const bool compute_dRdW, cons
     try {
 
         update_artificial_dissipation_discontinuity_sensor();
+        
+        //assembles and solves for auxiliary variable if necessary.
+        assemble_auxiliary_residual();
+
+//for(unsigned int i=0; i<auxiliary_solution[0].size(); i++){
+//auxiliary_solution[0].zero_out_ghosts();
+//pcout<<" aux "<<auxiliary_solution[0][i]<<std::endl;
+//}
```

Comment:
remove these commented lines

Reply:
fixed

Reply:
?

Reply:
Sorry I had that block in 2 places from debugging and I deleted the other block. I have just deleted this and now it's fixed.